source/jsonnet/england-wales/household/blocks/who-lives-here/do_you_usually_live_here.jsonnet

```diff
@@ -1,15 +1,15 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
 
 {
-  id: 'primary-person-list-collector',
+  id: 'do-you-usually-live-here',
   type: 'PrimaryPersonListCollector',
   for_list: 'household',
   add_or_edit_answer: {
-    id: 'you-live-here-answer',
+    id: 'do-you-usually-live-here-answer',
     value: 'Yes, I usually live here',
   },
   add_or_edit_block: {
-    id: 'add-or-edit-primary-person',
+    id: 'what-is-your-name-question',
```

Comment:
I think the block here should be `what-is-your-name` and the question beneath `what-is-your-name-question`?